CoroutineTemplate/data/build.gradle.kts

```diff
@@ -19,11 +19,14 @@ android {
 
     buildTypes {
         getByName(BuildType.RELEASE) {
-            isMinifyEnabled = false
-            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+            isMinifyEnabled = true
+            proguardFiles(
+                getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+            )
```

Comment:
@Tuubz please also take care of adding a new ticket to handle proguard setup, such as:
- keep Android stuff
- keep retrofit, moshi, okhttp
- keep API data class, enum, Parcelable
- keep logs, etc

Reply:
@luongvo Hmmmm, do we really need it though? In the screen-recording above, I built a release variant. The only update which was required to run successfully, was to add `@Keep` to `SecondBundle.kt`. So I don't think it's necessary to make an update for sample code.

Reply:
I'll test it out again asap, I think you might be right 👍

Reply:
Hmmmm, I tried accessing the Jetpack Compose screen from the app (it calls the API)

It's still working fine for me, not sure if I missed something. Would you mind creating an RFC for this then? 🙇